### PR TITLE
Feature/792 email templates ala

### PR DESCRIPTION
**Pull request by oerp-odoo**

### CODE REVIEW

File: account_mail_templates/templates/account_mail_templates.xml [hunk 1]
```diff
@@ -7,34 +7,32 @@
...
+                t-value="format_amount(record.amount_total, record.currency_id)"
+            />
+            <t t-set="extra_description_el">
+                <span
+                    name="span_paid_text"
+                    t-if="record.state == 'paid'"
+                >
+                    This invoice is already paid.
+                </span>
+                <span name="span_not_paid_text" t-else="">
```
Comment (SButko) [inline account_mail_templates/templates/account_mail_templates.xml:30]:
```suggestion
                <span 
                    name="span_not_paid_text" 
                    t-else=""
                >
```

File: base_common_note/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+{
+    'name': "Common Note",
+    'version': '12.0.1.0.0',
+    'summary': 'Common note that can use company data.',
+    'license': 'OEEL-1',
+    'author': "Focusate",
+    'website': "http://www.focusate.eu",
+    'category': 'Extra Tools',
+    'depends': [
+        'base'
```
Comment (SButko) [inline base_common_note/__manifest__.py]:
```suggestion
        # odoo
        'base'
```

**Review by SButko — APPROVED** [review]

File: account_mail_templates/templates/account_mail_templates.xml (hunk 1 above)
Comment (oerp-odoo) [reply account_mail_templates/templates/account_mail_templates.xml:30]: We indentate like that, when line becomes too long. Does not mean, always.

Comment (SButko) [reply account_mail_templates/templates/account_mail_templates.xml:30]: I know :) condition above seemed similar, so suggested to use same formatting.

**Pull request merged**